setup.py

```diff
@@ -34,7 +34,7 @@
       install_requires=['python-slimta',
                         'celery'],
       tests_require=['nose',
-                     'mox'],
+                     'mox3'],
```

Comment:
Going forward I'd like to remove the `tests_require` bit and replace it with a `test/requirements.txt` file. Mind doing that? I just made sure Jenkins will pick it up.